test/game_statistics_test.rb

```diff
@@ -25,18 +25,27 @@ def setup
 
   def test_it_exists
     game_statistics = GameStatistics.new(@game.stat_hash)
-
     assert_instance_of GameStatistics, game_statistics
   end
 
-  def test_highest_total_score
+  def test_it_has_a_total_score
     game_statistics = GameStatistics.new(@game.stat_hash)
+    assert_equal 3, game_statistics.total_goals[2]
+  end
 
+  def test_highest_total_score
+    game_statistics = GameStatistics.new(@game.stat_hash)
     assert_equal 5, game_statistics.highest_total_score
   end
 
   def test_lowest_total_score
     skip
     assert_equal 1, game_statistics.lowest_total_score
   end
+
+  def test_it_can_determine_percentage_of_home_wins
```

Comment:
Thanks for writing a test for your home_wins method. I like that you calculated the amount from the dummy file first. Do we need to consider a negation or a nil return and how to avoid it?